clio-client/src/main/scala/org/broadinstitute/clio/client/util/IoUtil.scala

```diff
@@ -62,54 +59,12 @@ trait IoUtil {
     gsUtil.exists(path.toString) == 0
```

Comment:
Too far away to comment on directly, but I'd also want to change `copyGoogleObject` and `deleteGoogleObject` to return `Unit` instead of `Int`, now that we can report failure through meaningful exceptions